src/syft/core/nodes/common/node.py

```diff
@@ -318,5 +322,15 @@ def _register_services(self) -> None:
         # of a sub-class of Node.
         self.services_registered = True
 
+    @syft_decorator(typechecking=True)
+    def recv_broadcast_msg(self, msg: SyftBroadcastMessage) -> None:
+        print("got a broadcast message")
+        return
+
+    @syft_decorator(typechecking=True)
```

Comment:
i don't think we should have any "listen" type message in Node. Grid should subclass and have all the methods like this